src/rust/dbv2/migrations/2024-02-07-144601_fees/up.sql

```diff
@@ -0,0 +1,33 @@
+-- Your SQL goes here
+CREATE TABLE aggregator.fees (
+    "time" TIMESTAMPTZ,
+    "market_id" NUMERIC(20,0),
+    "amount" NUMERIC(39,0),
```

Comment:
How about `"fees_in_quote_subunits"`, since plenty of DEXes, e.g. CPMMs charge in both base and quote?

Reply:
:+1: